js/interface.js

```diff
@@ -1799,7 +1811,7 @@ function configureAddRuleUI(rule) {
     showAutocompleteOnFocus: true
   });
 
-  $('input[name="exclude"]').tokenfield('setTokens', rule.exclude || []);
+  $('input[name="exclude"]').tokenfield('setTokens', rule.exclude ? rule.exclude : rule.include ? rule.include : []);
```

Comment:
ESLint does not allow nested ternary operations.

Reply:
@squallstar changes done as per suggestion please review